### ISSUE TITLE
[BITV] 9.4.1.2/9.5 - The button to display a version number should be implemented as a button element with an aria-expanded attribute. (1)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/documentation#9747]
- [ ] Fix
- [ ] Backport

![a65c5ec18479131a57c510247d533b19](https://user-images.githubusercontent.com/1315170/223755314-ef1b4dbc-908b-4300-8bb7-29fc02f3de9d.png)

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-bfbf07e5c3-v9-n5

</details>
